services/core-web/src/components/Forms/noticeOfWork/VerifyNoWContacts.js

```diff
@@ -108,10 +109,9 @@ const renderContacts = ({
               </p>
             </Col>
             {fields.map((field, index) => {
-              // eslint-disable-next-line no-unused-expressions
-              fields.get(index).id
-                ? (fields.get(index).id = fields.get(index).id)
-                : (fields.get(index).id = uuidv4());
+              if (!fields.get(index).id) {
```

Comment:
eslint was pretty angry about this bit of code and I also didn't like the use of ternary here.